### DISCUSSION TITLE
What is the process for finding a new device

### DISCUSSION THREAD

**Discussion by john55021** [zwave-js/zwave-js-ui#2096] [Q&A]
I have received a new 0x----.json file from a manufacturer and placed this file within the devices/0x---- folder for that manufacturer. I also added the appropriate entries into the index.json file. After  several re-interview attempts, the device info is never found. I read somewhere that devices have to be within https://devices.zwave-js.io/  to obtain the product information (this device is new so isn't in it). Is this the case?

**Comment by robertsLando** [comment]
https://zwave-js.github.io/zwavejs2mqtt/#/device-support/custom-device-files

**Reply by john55021** [reply]
I must be so lost on this docker-compose stuff. I created a folder config within /home/pi/store and placed the .json file in it. Nothing appeared on the screen after docker-compose up indicting it picked it up. So I turned on logging to a file within the zwavejs2mqtt interface. And, when going to the folder icon on the left size of the interface, a log file named zwavejs2mqtt_12.....log is there. However, when I search for the log file within the Linux file manager, nothing is there. I likewise create a file thru the interface, and that file doesn't seem to exist within the Linux file system. So it certainly appears that the /home/pi/store is not even used. Where is everything???? And how do you get to it to do things like add a folder????

**Reply by robertsLando** [reply]
Show me your docker compose

**Comment by john55021** [comment]
Its exactly the one online except the device name. Using the file system tree within zwavejs2mqtt in able to create folders and files.  So i tried that, created a config file and added the 0x1234.json file and it worked. So docker must create its own internal folder system as opposed to using Linux. Is there a way just to use the Linux system instead? That is if my supposition is correct.Thanks JohnSent via the Samsung Galaxy S10, an AT&T 5G Evolution capable smartphone
-------- Original message --------From: Daniel Lando ***@***.***> Date: 12/27/21  2:51 AM  (GMT-06:00) To: zwave-js/zwavejs2mqtt ***@***.***> Cc: john55021 ***@***.***>, Author ***@***.***> Subject: Re: [zwave-js/zwavejs2mqtt] What is the process for finding a new device (Discussion #2096) 
Show me your docker compose
